test/e2e/specs/web_app.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  'Demo WebbApp test': function (browser) {
+    browser
+      .url('https://repartyed.firebaseapp.com')
+      .waitForElementVisible('body', 5000)
+
+      .waitForElementVisible('#app > div.application--wrap > div > main > div.container.fluid.fill-height > div > div > div > div > div > div.card__actions > button > div', 5000)
+      .click('#app > div.application--wrap > div > main > div.container.fluid.fill-height > div > div > div > div > div > div.card__actions > button > div')
+      // .assert.containsText('#main', 'Night Watch')
```

Comment:
If this isn't being used we should remove it.
We don't want to be pushing these types of comments up to dev or master.